.gitignore

```diff
@@ -8,3 +8,4 @@ pkg/contracts
 
 /tellor
 
+/configs/myconfig*
```

Comment:
This looks specific to a local setup so best to keep these excludes in

`.git/info/exclude`
this file it not tracked and can put all excludes specific to someones local env setup.